static/js/compose.vue

```diff
@@ -199,6 +199,14 @@
         });
       }, 200),
       submit: function(){
+        if (this.userrequest.observation_type === 'TARGET_OF_OPPORTUNITY'){
+          for (var index = 0; index < this.userrequest.requests.length; ++index) {
+            for (var windowIndex = 0; windowIndex < this.userrequest.requests[index].windows.length; ++windowIndex) {
+              this.userrequest.requests[index].windows[windowIndex].start = new Date($.now());
```

Comment:
This should probably be done in the serializer, so that it can apply to api submissions as well as UI submissions

Reply:
Also hackerz